.sass-lint.yml

```diff
@@ -0,0 +1,182 @@
+# sass-lint config generated by make-sass-lint-config v0.1.2
```

Comment:
Did you decide that sass-lint has advantages over StyleLint?
If so, can you describe them?